src/tribol/geom/ContactPlane.cpp

```diff
@@ -2148,8 +2157,11 @@ void ContactPlane2D::computeAreaTol()
 {
    // Note: this code is the same as for ContactPlane3D, but maintain separate 
    // routine for 2D treatment.
-   if (m_areaFrac < 1.E-12) {
-      SLIC_ERROR ( "ContactPlane2D::computeAreaTol the overlap area fraction too small or negative" );
+   parameters_t & parameters = parameters_t::getInstance();
+
+   if (m_areaFrac < parameters.overlap_area_frac ) {
+      SLIC_DEBUG( "ContactPlane2D::computeAreaTol() the overlap area fraction too small or negative; " << 
+                  "setting to overlap_area_frac parameter." );
```

Comment:
Minor: Can/should we use `SLIC_DEBUG_IF` here instead?

Reply:
I actually need to make this consistent with the 3D case where I reset the area frac. I updated this.